main.go

```diff
@@ -246,6 +258,13 @@ func main() {
 			fmt.Printf("%#v\n", err)
 			os.Exit(65)
 		}
+	case arguments["new-openpgp-keypair"]:
+		err := generateOpenPGPKeypair(arguments["--file"].(string))
+		if err != nil {
+			fmt.Println("Error generating OpenPGP keypair for worker:")
+			fmt.Printf("%#v\n", err)
+			os.Exit(66)
```

Comment:
I started exit codes at 64, and increased with each one I added. See http://tldp.org/LDP/abs/html/exitcodes.html - in particular:

> The author of this document proposes restricting user-defined exit codes to the range 64 - 113 (in addition to 0, for success), to conform with the C/C++ standard.